recipes/protobuf/all/conanfile.py

```diff
@@ -92,6 +92,16 @@ def _patch_sources(self):
 endif()
 """
 
+        if not self.options.with_rtti:
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, "cmake", "CMakeLists.txt"),
+                "if(WITH_PROTOC)",
```

Comment:
You could add a cmake option and `add_definitions` in the "Conan CMake wrapper" https://github.com/conan-io/conan-center-index/blob/master/recipes/protobuf/all/CMakeLists.txt#L3
I think this would be cleaner then a replace_in_file.

Getting it up streamed is even better though :)

Reply:
I'm on it!